packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -813,11 +813,18 @@ export const prepareVaultManagerKit = (
             Fail`Must have locked a quote before liquidating vaults.`;
           assert(lockedQuote); // redundant with previous line
 
-          const crKey = normalizedCollRatioKey(lockedQuote, compoundedInterest);
+          const liqMargin = facets.self
+            .getGovernedParams()
+            .getLiquidationMargin();
+          const crKey = normalizedCollRatioKey(
```

Comment:
Investigating this change made me realize: vaultManager shouldn't be concerned with the encoding. Even liquidation.js shouldn't.

`vaultManager` is concerned with the parameters
`liquidation` turns those into a collateralization ratio threshold
`prioritizedVaults` takes that ratio and uses the encoding utils to query its store